proj_utils/Parser.py

```diff
@@ -60,7 +60,7 @@ def parse_isgoodbehaviour(self):
         return False
 
     def create_text_file(self):
-        save_path = 'C:/Users/razm1/PycharmProjects/routeExercise/students'
+        save_path = 'C:/Users/razm1/PycharmProjects/routeExerciseWTesting/students'
```

Comment:
you changed your code for your tests? 
that shouldnt happend, 
your test should write to a different environment (i.g = C:/Users/razm1/PycharmProjects/routeExerciseTest/students_test)
this means its a different directories, that indicative.
the path shouldnt be hardcoded. it should be configurable (when prod -> writes to 1 path. when test -> different path) 

also, why are you using an absolute path and not relative path?